source/testnet/guides/become-baker.rst

```diff
@@ -99,15 +76,13 @@ Stake and lottery
 -----------------
 
 .. todo::
-
-   - Clarify what it means that the amount has to be *released* (first we need to say that
-     the amount needs to staked)
+   
    - Link to release schedule.
-   - Reword "*holds* *some* stake".
-
-A baker holds some *stake* which is part of the amount of GTU owned by the
-account. The staked amount cannot be moved or transferred until it
-is released by the baker.
+     
+The account can provide part of its owned amount of GTU to the baker (amount to
+which we will then refer as *staked amount*) and can later manually release all
```

Comment:
I don't think convert is the right word. It implies some action that does not exist, I think.

I think we use  `stake` as a verb to describe the action.

Reply:
I was thinking of using another verb because stake is introduced for the first time here, and I thought using "convert" would make it more gradual :) The reason I chose "convert" is that it also emphasizes that you take part of your balance and move it into another state which somewhat suggests that it's not part of your normal balance any more. By contrast, the verb "stake" might only be clear once you know what it means.

Reply:
Using `stake` as a verb.

Reply:
But we shouldn't say "stake GTU into baker stake". I'm OK with not using "convert" but I don't think we should use "stake" both as a verb and direct object in the same sentence. @abizjak could you provide a suggestion that you think would fit?

Reply:
If you feel strongly about this I'm fine if you use convert. I don't have time to rephrase at the moment.